src/superqt/collapsible/_collapsible.py

```diff
@@ -11,16 +20,16 @@ class QCollapsible(QFrame):
     Based on https://stackoverflow.com/a/68141638
     """
 
-    _EXPANDED = "▼  "
-    _COLLAPSED = "▲  "
-
     def __init__(self, title: str = "", parent: Optional[QWidget] = None):
```

Comment:
This should maybe also expose the `collapsedIcon` and the `expandedIcon` as Optional arguments (as some buttons allow in Qt)?

What do you think @tlambert03 ?

Reply:
I added this.